src/cf-forms/src/molecules/form-fields.less

```diff
@@ -101,4 +101,73 @@
             }
         }
     }
+
+    &__select {
+        .a-select {
+            position: relative;
+            margin-top: 0.5em;
+            border: 1px solid @select-border;
+
+            select {
+                height: unit( 30px / @base-font-size-px, em );
+                width: 100%;
+                padding: unit( 4px / @base-font-size-px, em )
+                         unit( 6px / @base-font-size-px, em );
+                border: 0;
+                outline: 2px solid transparent;
+                appearance: none;
+                background-color: @input-bg;
+                border-radius: 0;
+                color: @text;
+
+                &:hover,
+                &:active,
+                &:focus {
+                    outline-color: @input-border__focused;
+                    outline-offset: 0;
+                }
+            }
+
+            select[disabled] {
+                color: @input-text__disabled;
+                background-color: @input-bg__disabled;
+                cursor: not-allowed;
+
+                &:hover {
+                    outline: 2px solid transparent;
```

Comment:
We [use it in cf-tables](https://github.com/cfpb/capital-framework/blob/8b2807216061f88f5930040482f4c2fcfa1951a4/src/cf-tables/src/cf-tables.less#L54) and outline shouldn't be moving elements, it doesn't affect document flow. From https://developer.mozilla.org/en-US/docs/Web/CSS/outline

> Outlines do not take up space, they are drawn above the content.